docs/HACKING.md

```diff
@@ -1,26 +1,30 @@
 # Goldstone Server Hacking Guide
 
 
-This explains how to install and run Goldstone Server locally (mostly in docker containers), so you can do code development on the project.  The instructions assume a Mac OS X Yosemite development environment with [homebrew](http://brew.sh/) and [Docker Toolbox](http://www.docker.com/toolbox) installed.  
+This explains how to install and run Goldstone Server locally, so you can do code development on the project.  These instructions are written for a Mac OS X Yosemite development.
 
-**_Note: if you have manually installed docker-machine and docker-compose, make sure your docker-machine VM name is 'default' in order to be compatible with the supporting scripts._**
 
 [TOC]
 
 ## Prerequisites
 
-Install various prerequisite packages:
+Install [Docker Toolbox](https://github.com/docker/toolbox/releases).  ** There is an issue with release 1.8.2a, but earlier versions work **
```

Comment:
Everywhere else in this file, statements of note are bolded and italicized.  It should be the same here.